content/en/docs/contribute/_index.md

```diff
@@ -145,27 +145,26 @@ class first,second white
 {{</ mermaid >}}
 Figure 2. Preparation for your first contribution.
 
-- Read the [Contribution overview](/docs/contribute/new-content/) to
+1. Read the [Contribution overview](/docs/contribute/new-content/) to
```

Comment:
This is the recommended pathway, however is not the ONLY way to prepare for your first contribution.

Reply:
Please take a look at the mermaid diagram

Firstly, I added `(Optional)` for first thee, but, then I realize that it would not help at all. 

So, in the live version diagram or list is wrong.

Reply:
The diagram is right and the unordered list doesn't invalidate that.